setup.py

```diff
@@ -21,6 +22,29 @@
     if start_pos >= 0:
         long_description = long_description[start_pos:]
 
+install_requires = ["numpy>=1.15", "onnxconverter-common>=1.6.0", "scikit-learn==0.22.1"]
+if sys.platform == "darwin" or sys.platform == "linux":
+    install_requires.append("torch>=1.4.0")
+else:
+    if sys.version_info >= (3, 5, 0):
```

Comment:
can we use the version from the github link you posted? not with dependency links.

Reply:
This try? 
https://github.com/microsoft/hummingbird/pull/128/commits/32cfc3b3be4fab4e5302acac0a1ddf6446ab1a5d
?

Reply:
I think should be this `sys.version_info[:2] == (3, 5):`

Reply:
trying.....